specification/web/resource-manager/Microsoft.Web/stable/2023-01-01/ResourceProvider.json

```diff
@@ -507,6 +507,53 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/locations/{location}/quotaType/{quotaType}/usages": {
+      "get": {
+        "summary": "Gets subscription core usages for location, for a given type of quota.",
```

Comment:
Consider rephrasing to "Gets the list of"?
At least, this ARM URI path looks intended to query for a list, according to ARM conventions.